src/index.html

```diff
@@ -1,13 +1,368 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link
+      type="Image/x-icon"
+      href="images/favicon.png"
+      rel="icon"
+    >
+    <title>
+      MyBike
+    </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <script src="scripts/main.js" defer>
+
+    </script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header" >
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top top-actions">
+            <a href="/" class="logo">
+              <img
+              class="logo__image"
+              src="../src/images/MyBiKE.svg" alt="MyBike">
+            </a>
+            <div class="top-actions">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone header__call">
+              </a>
+              <a
+              href="#menu"
+              class="icon icon--menu"
+            >
+              </a>
+            </div>
+          </div>
+          <h1 class="header__tittle">
+          Take the Streets
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu" >
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+              class="logo__image"
+              src="../src/images/MyBiKE.svg" alt="MyBike">
+            </a>
+            <a href="#" class="icon icon--cross">
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">
+                Home
+              </a>
+            </li>
+            <li class="menu__item">
+              <a href="#about_us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compare_bikes" class="menu__link">Compare Bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+          <a
```

Comment:
Maube it will lok good, if you will add some hover effect also for this link in menu